roles/ovirt-engine-setup/README.md

```diff
@@ -0,0 +1,109 @@
+oVirt Engine Setup
+==================
+
+Generate answerfile and run engine-setup with it. In case you pass variable
+'ovirt_engine_update: True' also setup packages will be updated.
+
+Target Systems
+--------------
+
+* engine
+
+Requirements
+------------
+
+ * Preinstalled clean environment with configured repositories.
+ * Ansible version 2.3
+
+Role Variables
+--------------
+
+By default engine-setup uses answer file specific for version of oVirt,
+based on ``ovirt_engine_version`` parameter. You can specify own answer file
+as ``ovirt_engine_answer_file_path``.
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_type               | ovirt-engine          | Type of product 'ovirt-engine' or 'rhevm'  |
```

Comment:
should the product be 'rhvm' instead of 'rhevm'

Reply:
should the product be 'rhv' instead of 'rhvm'

Reply:
I think this parameter can be removed. we know the version, we can create the matrix for package name per version and hide it from user.

Reply:
Yes, but how do you know if user wants to install oVirt or RHV?

Reply:
Hmm, you 're right. I 'll keep it